cmd/glasskube/cmd/configure.go

```diff
@@ -74,28 +83,20 @@ func runConfigure(cmd *cobra.Command, args []string) {
 		fmt.Fprintf(os.Stderr, "⚠️  Some values can not be resolved: %v\n", err)
 	}
 
-	if !cliutils.YesNoPrompt("Continue?", true) {
-		cancel()
+	if !configureCmdOptions.DryRun {
+		if !cliutils.YesNoPrompt("Continue?", true) {
+			cancel()
+		}
 	}
 
 	switch pkg := pkg.(type) {
 	case *v1alpha1.ClusterPackage:
-		if err := pkgClient.ClusterPackages().Get(ctx, pkg.Name, pkg); err != nil {
-			// Don't exit, we can still try to call update ...
-			fmt.Fprintf(os.Stderr, "⚠️  error fetching package: %v\n", err)
-		}
-
-		if err := pkgClient.ClusterPackages().Update(ctx, pkg); err != nil {
+		if err := pkgClient.ClusterPackages().Update(ctx, pkg, opts); err != nil {
```

Comment:
Thank you for finding and fixing this bug, however the reason why we fetched the package again was that it might take the user some time to complete the configuration wizard, so the chance that the resource would have been updated in the meantime is relatively high. This leads to an error because k8s only allows updating on the most recent resource version. 

Hence, I think something like this would be a better fix:
```go
values := pkg.Spec.Values
if err := pkgClient.ClusterPackages().Get(ctx, pkg.Name, pkg); err != nil {
	// Don't exit, we can still try to call update ...
	fmt.Fprintf(os.Stderr, "⚠️  error fetching package: %v\n", err)
}
pkg.Spec.Values = values
```

Reply:
I see @kosmoz thanks for the explanation, I'll proceed that way